CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/display/GraphVisualAccess.java

```diff
@@ -23,6 +23,7 @@
 import au.gov.asd.tac.constellation.graph.utilities.ConnectionMode;
 import au.gov.asd.tac.constellation.graph.visual.color.ColorAttributeDescription;
 import au.gov.asd.tac.constellation.graph.visual.concept.VisualConcept;
+import au.gov.asd.tac.constellation.preferences.GraphPreferenceKeys;
```

Comment:
Is this an unused import?

Reply:
See latest commit "Resolve PR changes" Yes, It is.